tests/features/event.feature

```diff
@@ -218,6 +228,11 @@ Feature: Event content creation
     And I should see the text "Rue belliard 28"
     And I should see the text "1000 Brussels"
     And I should see the text "Belgium"
+    # Event partner values.
+    And I should see the text "Name of the event partner"
+    And I should see an "AV portal photo of the Event partner" element
```

Comment:
Should we not check that the image is there, not just the wrapper div?

Reply:
Not yet, the theming ticket is still to be done. There we will take care of checking these. This is just a smoke test, that the field is actually output in any way.